src/components/YoutubeList/YoutubeListItem.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { useContext } from 'react';
+import { Card } from 'react-bootstrap';
+import { useHistory } from 'react-router';
+import GlobalContext from '../../state/GlobalContext';
+
+function YoutubeListItem(props) {
+  const history = useHistory();
+  const { setActiveVideo } = useContext(GlobalContext);
+  const handleClick = () => {
+    setActiveVideo({
+      videoId: props.videoId,
+      title: props.title,
+      description: props.description,
+      channelTitle: props.channelTitle,
+      imageUrl: props.imageUrl,
+    });
+    if (history.location.pathname === '/') {
+      history.push(`/video/${props.videoId}`);
+    } else if (history.location.pathname === '/favorites') {
+      history.push(`/favorites/${props.videoId}`);
+    }
+  };
+
+  return (
+    <Card
+      style={{ width: '20rem', height: '450px' }}
+      className="card-click"
+      onClick={handleClick}
+    >
+      <Card.Img variant="top" src={props.imageUrl} />
+      <Card.Body>
+        <Card.Title>{props.title}</Card.Title>
```

Comment:
For some reason, the title of the Card contains some [HTML Entities](https://www.w3schools.com/html/html_entities.asp). Try to escape these characters.
<img width="352" alt="Screen Shot 2020-12-09 at 18 44 42" src="https://user-images.githubusercontent.com/6569533/101710414-aaa91580-3a4e-11eb-84d2-0bf559477611.png">

Reply:
It's working now